Description: ijs_server: Don't ignore the 'sign' var when parsing ints
Origin: upstream, http://git.ghostscript.com/?p=ghostpdl.git;h=882ce8b
Author: Robin Watts <robin.watts@artifex.com>
Forwarded: yes
Last-Update: 2019-08-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/ijs_server.c
+++ b/ijs_server.c
@@ -504,7 +504,7 @@
 	return IJS_ESYNTAX;
       num = (num * 10) + (c - '0');
     }
-  *result = num;
+  *result = sign * num;
   return 0;
 }
 
